eng/Packages.Data.props

```diff
@@ -128,6 +128,8 @@
     <PackageReference Update="Azure.Storage.Blobs" Version="12.21.1" />
     <PackageReference Update="Azure.Storage.Queues" Version="12.19.1" />
     <PackageReference Update="Azure.Storage.Files.Shares" Version="12.19.1" />
+    <PackageReference Update="Azure.Search.Documents" Version="11.6.0" />
+    <PackageReference Update="Azure.AI.Projects" Version="1.0.0-beta.2" />
```

Comment:
Can we please move the AI libraries up in the list to preserve alphabetical ordering?